CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * [#6856](https://github.com/rubocop-hq/rubocop/pull/6856): Fix auto-correction for `Style/BlockComments` when the file is missing a trailing blank line. ([@ericsullivan][])
 * [#6858](https://github.com/rubocop-hq/rubocop/issues/6858): Fix an incorrect auto-correct for `Lint/ToJSON` when there are no `to_json` arguments. ([@koic][])
 * [#6865](https://github.com/rubocop-hq/rubocop/pull/6865): Fix deactivated `StyleGuideBaseURL` for `Layout/ClassStructure`. ([@aeroastro][])
+* [#6869](https://github.com/rubocop-hq/rubocop/pull/6869): Fix false positive for `Rails/LinkToBlank`. ([@r7kamura][])
```

Comment:
Can you write a little more about the reason for false positive? It will be user friendly 😃

Reply:
Right. Slightly modified like this:

```diff
- Fix false positive for `Rails/LinkToBlank`.
+ Fix false positive for `Rails/LinkToBlank` when rel is a symbol value.
```